static/_redirects

```diff
@@ -130,6 +130,7 @@
 /docs/concepts/scheduling-eviction/eviction-policy/     /docs/concepts/scheduling-eviction/node-pressure-eviction/ 301
 /docs/concepts/service-catalog/     /docs/concepts/extend-kubernetes/service-catalog/ 301
 /docs/concepts/services-networking/networkpolicies/     /docs/concepts/services-networking/network-policies/ 301
+/zh/docs/concepts/services-networking/add-entries-to-pod-etc-hosts-with-host-aliases/ /zh/docs/tasks/network/customize-hosts-file-for-pods/ 301
```

Comment:
I'm not convinced that we do need to add such an redirect per language. In theory, this is the right thing to do. In practice, however, I don't see the possibility people have bookmarks for this page.
So ... what I'd suggest is that we intentionally create a "bug" here. If there are complaints about the missing redirect, we fix it. Otherwise, we don't flood the redirects file with this kind of changes.

Reply:
I agree it's unlikely the redirect is needed. I also double checked the original url is not referenced by our internal documents. So I removed the change in the redirect file. Thanks.